src/core/bitop.d

```diff
@@ -72,27 +72,35 @@ unittest
  */
 int bsf(uint v) pure
 {
-    static if (size_t.sizeof == ulong.sizeof)
+    pragma(inline, false);  // so intrinsic detection will work
+    static if (size_t.sizeof == ulong.sizeof)  // 64 bit code gen
```

Comment:
I originally used a `static if` here because `bsf(uint)` was not an intrinsic on 64-bit, and the **intrinsic** `bsf(ulong)` is faster than `softBsf!uint()`.

However, the software fallback `softBsf!ulong()` is slower, so you should ditch the `static if` and just always call `softBsf!uint()` for the fallback.